client/rpc/src/state/state_light.rs

```diff
@@ -339,6 +340,14 @@ impl<Block, F, Client> StateBackend<Block, Client> for LightState<Block, F, Clie
 		Box::new(result(Err(client_err(ClientError::NotAvailableOnLightClient))))
 	}
 
+	fn proof(
+		&self,
+		_block: Block::Hash,
+		_keys: Vec<StorageKey>,
+	) -> FutureResult<StorageProof> {
+		Box::new(result(Err(client_err(ClientError::NotAvailableOnLightClient))))
```

Comment:
@svyatonik does it makes sense here to forward that request to the connected full node?

Reply:
This is doable, but I'm not sure if light client should have this functionality exposed. Light client shouldn't serve as proof server imo.

Reply:
Okay, so we stick to `NotAvailable` :)